Lsof.8

```diff
@@ -4457,6 +4453,7 @@ is the first component of the host's name returned by
 .SH AUTHORS
 .I Lsof
 was written by Victor A.\&Abell <abe@purdue.edu> of Purdue University.
+Since version 4.93.0, the lsof-org team at GitHub maintains.
```

Comment:
I talked to my wife who is an editor and great person on grammar.  
Ending the sentence with maintains.  makes it a question: (maintains what?).

Can we add lsof between the maintains and the period?

Reply:
Oh, I see.

Reply:
I do understand English grammar is hard, and definitely not your first language. I'm sure I'd make a mess of Japanese.